jinja2/base.html

```diff
@@ -158,10 +166,14 @@
     {% javascript 'main' %}
     <script>
     if (window.mdn && mdn.analytics) mdn.analytics.trackOutboundLinks();
-</script>
+  </script>
     {% for script in scripts %}
       {% javascript script %}
     {% endfor %}
+    {% if settings.MDN_CONTRIBUTION %}
+    <script src="https://checkout.stripe.com/checkout.js"></script>
```

Comment:
We'd like to lazy-load this. I thought I read something in the Stripe docs that recommended the opposite, including the code on each page for fraud detection. I'm not finding it on the Checkout help page, maybe that was for a different integration.

Adding ``async`` may help, but we may get into a situation where ``StripeCheckout`` isn't available yet.

Here's the custom integration docs, which appear to be the model for the code:

https://stripe.com/docs/checkout#integration-custom